src/krisi/evaluate/library/metric_wrappers.py

```diff
@@ -114,8 +116,8 @@ def pred_y_imbalance_ratio(
     pos_label: int = 1,
     **kwargs,
 ) -> float:
-    prediction_frequencies = preds.value_counts().dropna()
-    target_frequencies = y.value_counts().dropna()
+    prediction_frequencies = preds.value_counts(normalize=True).dropna()
+    target_frequencies = y.value_counts(normalize=True).dropna()
     if len(prediction_frequencies) == 1 or len(target_frequencies) == 1:
         logger.warning("Only one class in either predictions or target, returning 0")
```

Comment:
I think we should take this out, as it creates a log every time during the rolling evaluation